src/twisted/application/runner/_exit.py

```diff
@@ -66,72 +40,60 @@ class Status:  # type: ignore[no-redef]
-    @type EX_DATAERR: L{ValueConstant}
-
     @cvar EX_NOINPUT: Cannot open input.
-    @type EX_NOINPUT: L{ValueConstant}
-
     @cvar EX_NOUSER: Addressee unknown.
-    @type EX_NOUSER: L{ValueConstant}
-
     @cvar EX_NOHOST: Host name unknown.
-    @type EX_NOHOST: L{ValueConstant}
-
     @cvar EX_UNAVAILABLE: Service unavailable.
-    @type EX_UNAVAILABLE: L{ValueConstant}
-
     @cvar EX_SOFTWARE: Internal software error.
-    @type EX_SOFTWARE: L{ValueConstant}
-
     @cvar EX_OSERR: System error (e.g., can't fork).
-    @type EX_OSERR: L{ValueConstant}
-
     @cvar EX_OSFILE: Critical OS file missing.
-    @type EX_OSFILE: L{ValueConstant}
-
     @cvar EX_CANTCREAT: Can't create (user) output file.
-    @type EX_CANTCREAT: L{ValueConstant}
-
     @cvar EX_IOERR: Input/output error.
-    @type EX_IOERR: L{ValueConstant}
-
     @cvar EX_TEMPFAIL: Temporary failure; the user is invited to retry.
-    @type EX_TEMPFAIL: L{ValueConstant}
-
     @cvar EX_PROTOCOL: Remote error in protocol.
-    @type EX_PROTOCOL: L{ValueConstant}
-
     @cvar EX_NOPERM: Permission denied.
-    @type EX_NOPERM: L{ValueConstant}
-
     @cvar EX_CONFIG: Configuration error.
-    @type EX_CONFIG: L{ValueConstant}
     """
 
-    EX_OK = ValueConstant(Status.EX_OK)
-    EX_USAGE = ValueConstant(Status.EX_USAGE)
-    EX_DATAERR = ValueConstant(Status.EX_DATAERR)
-    EX_NOINPUT = ValueConstant(Status.EX_NOINPUT)
-    EX_NOUSER = ValueConstant(Status.EX_NOUSER)
-    EX_NOHOST = ValueConstant(Status.EX_NOHOST)
-    EX_UNAVAILABLE = ValueConstant(Status.EX_UNAVAILABLE)
-    EX_SOFTWARE = ValueConstant(Status.EX_SOFTWARE)
-    EX_OSERR = ValueConstant(Status.EX_OSERR)
-    EX_OSFILE = ValueConstant(Status.EX_OSFILE)
-    EX_CANTCREAT = ValueConstant(Status.EX_CANTCREAT)
-    EX_IOERR = ValueConstant(Status.EX_IOERR)
-    EX_TEMPFAIL = ValueConstant(Status.EX_TEMPFAIL)
-    EX_PROTOCOL = ValueConstant(Status.EX_PROTOCOL)
-    EX_NOPERM = ValueConstant(Status.EX_NOPERM)
-    EX_CONFIG = ValueConstant(Status.EX_CONFIG)
+    EX_OK = Status.EX_OK
+    EX_USAGE = Status.EX_USAGE
+    EX_DATAERR = Status.EX_DATAERR
+    EX_NOINPUT = Status.EX_NOINPUT
+    EX_NOUSER = Status.EX_NOUSER
+    EX_NOHOST = Status.EX_NOHOST
+    EX_UNAVAILABLE = Status.EX_UNAVAILABLE
+    EX_SOFTWARE = Status.EX_SOFTWARE
+    EX_OSERR = Status.EX_OSERR
+    EX_OSFILE = Status.EX_OSFILE
+    EX_CANTCREAT = Status.EX_CANTCREAT
+    EX_IOERR = Status.EX_IOERR
+    EX_TEMPFAIL = Status.EX_TEMPFAIL
+    EX_PROTOCOL = Status.EX_PROTOCOL
+    EX_NOPERM = Status.EX_NOPERM
+    EX_CONFIG = Status.EX_CONFIG
+
+
+def exit(status: Union[int, ExitStatus], message: Optional[str] = None) -> NoReturn:
+    """
+    Exit the python interpreter with the given status and an optional message.
+
+    @param status: An exit status. An appropriate value from L{ExitStatus} is
+        recommended.
+    @param message: An options message to print.
+    """
+    if message:
+        if status == 0:
```

Comment:
Maybe this is outside the scope of this PR, but wouldn't it be clearer to compare to `EX_OK` instead of 0?

Reply:
Yes, thanks